actionmailer/lib/action_mailer/base.rb

```diff
@@ -900,15 +913,19 @@ def collect_responses(headers)
           yield(collector)
           collector.responses
         elsif headers[:body]
-          [{
-            body: headers.delete(:body),
-            content_type: self.class.default[:content_type] || "text/plain"
-          }]
+          collect_responses_from_text(headers)
         else
           collect_responses_from_templates(headers)
         end
       end
 
+      def collect_responses_from_text(headers)
+        [{
+          body: headers.delete(:body),
+          content_type: headers[:content_type] || self.class.default[:content_type] || "text/plain"
```

Comment:
@sgrif should we consider to use different name here like `body_type` instead of `content_type`, because there are some cases mailer will have many parts like mailer `content_type`, body `content_type`